tests/model_options/test_tablespaces.py

```diff
@@ -133,3 +136,47 @@ def test_tablespace_for_many_to_many_field(self):
         # The ManyToManyField declares db_tablespace, its indexes go there.
         self.assertNumContains(sql, "tbl_tbsp", 0)
         self.assertNumContains(sql, "idx_tbsp", 2)
+
+
+class DefaultTablespaceTests(TransactionTestCase):
+    available_apps = ["model_options"]
+
+    def assertNumContains(self, haystack, needle, count):
+        real_count = haystack.count(needle)
+        self.assertEqual(
+            real_count,
+            count,
+            "Found %d instances of '%s', expected %d" % (real_count, needle, count),
+        )
+
+    @skipUnlessDBFeature("supports_tablespaces")
+    @isolate_apps("model_options")
+    def test_default_tablespace(self):
+        tablespace = "default_tablespace"
+        databases = copy.deepcopy(settings.DATABASES)
+        databases["default"]["OPTIONS"]["DEFAULT_TABLESPACE"] = tablespace
+        with self.settings(DATABASES=databases):
+
+            class ScientistRef(models.Model):
+                name = models.CharField(max_length=50)
+
+            sql = sql_for_table(ScientistRef).lower()
+
+        # 1 for the table + 1 for the index on the primary key
+        self.assertNumContains(sql, tablespace, 2)
+
+    @skipUnlessDBFeature("supports_tablespaces")
+    @isolate_apps("model_options")
+    def test_default_index_tablespace(self):
+        index_tablespace = "default_index_tablespace"
+        databases = copy.deepcopy(settings.DATABASES)
+        databases["default"]["OPTIONS"]["DEFAULT_INDEX_TABLESPACE"] = index_tablespace
```

Comment:
I could be wrong here but I _think_ these options should live at the same level as `OPTIONS` and not be nested within `OPTIONS`?

In other words, I think the setting should be defined as 

```
DATABASES = {
    "default": {
        "ENGINE": "...",
        ...,
        "DEFAULT_TABLESPACE": "..."
        "DEFAULT_INDEX_TABLESPACE": "...",
     }
}
```

As `OPTIONS` is meant to be per-backend flags that are usually passed directly to the underlying client bindings.

Reply:
Thanks, @charettes. The [ticket comment](https://code.djangoproject.com/ticket/20960#comment:2) seems to suggest they should be inside "OPTIONS", but I'm fine with whatever.  @aaugustin do you have any comments?

Reply:
Sorry, I haven't been involved in developing Django for several years now and I'm unable to help there.

Reply:
No problem, thanks.

I'll update the code to move these options up to the same level as `OPTIONS`.

Reply:
@charettes @felixxm @shangxiao I've updated the code to move those options up.